blsSignatures/blsSignatures.go

```diff
@@ -177,41 +192,42 @@ func VerifyAggregatedSignatureDifferentMessages(sig Signature, messages [][]byte
 //
 // If keyValidationMode is true, this uses a tweaked version of the padding,
 // so that the result will not collide with a result generated in an ordinary signature.
-func hashToG1Curve(message []byte, keyValidationMode bool) (*bls12381.PointG1, error) {
+func hashToG1Curve(message []byte, keyValidationMode bool) (*bls12381.G1Affine, error) {
```

Comment:
We should add a test case to this function with hardcoded inputs and outputs (generated from the previous version) to ensure that this new library's MapToG1 is the same as the old one (there's historically been different standards for this, though nowadays they should hopefully be canonical).

Reply:
Done. Covered in TestHashToG1Curve.